ci/jenkins/pipelines/skuba-e2e-daily.Jenkinsfile

```diff
@@ -61,7 +61,7 @@ pipeline {
             archiveArtifacts(artifacts: "skuba/ci/infra/${PLATFORM}/terraform.tfstate", allowEmptyArchive: true)
             archiveArtifacts(artifacts: "skuba/ci/infra/${PLATFORM}/terraform.tfvars.json", allowEmptyArchive: true)
             archiveArtifacts(artifacts: 'testrunner.log', allowEmptyArchive: true)
-            archiveArtifacts(artifacts: 'skuba/ci/infra/testrunner/*.xml', allowEmptyArchive: true)
```

Comment:
Why stop collecting the test results?

Reply:
The test results are not being generated because we don't run the tests without the junit option. We can introduce this later in another PR.